src/headlessMoneyHash.ts

```diff
@@ -504,13 +507,16 @@ export default class MoneyHashHeadless<TType extends IntentType> {
         });
 
         return {
-          mount: () =>
+          mount: () => {
+            this.mountedCardElements.push(elementType);
```

Comment:
Can we push this line after render, maybe the rendering fails, because of the css selector nor found

Reply:
we have a specific validation for the css selector so it won't be needed as if there is an issue with the selector it will be fired before this line

![Screenshot 2024-08-14 at 1 36 28 PM](https://github.com/user-attachments/assets/4ec40ff8-5ff6-4fea-911c-49e4d6e929bc)

Reply:
but the elementType will be pushed to the mounted ones but he didn't,

So logically this line should be at the end of the function

Reply:
oh ok sorry, my bad